### ISSUE TITLE
Unable to open UI after restoring from backup

### ISSUE THREAD

**Issue by da-kami** [itchysats/start9-itchysats-wrapper#7]
What did I do:

1. Install itchysats service from `itchysats.s9pk`
2. Create backup of itchysats service
3. Uninstall itchysats service
4. Restore from backup

The restore completes and the `/data` is properly added, i.e. the files are correct and the service starts up correctly (checked the logs). 
But the configuration does not seem to be correct, because the `LAUNCH UI` button points to an outdated / incorrect link. So I am unable to access the UI.

When `cat /etc/nginx/sites-enabled/itchysats_main_443.conf` I can see the correct address (through which I am able to access the UI). 

A restart *may* fix this problem, i.e. the link to the UI was correct afterwards, but I also had cases where it did not, i.e. the link remained wrong.

Todo: Understand if this is a problem of how the backup+restore is configured or a different issue.

**Comment by islandbitcoin** [comment]
have you tested to see if this is a local browser caching issue?

**Comment by da-kami** [comment]
Browser: Firefox 

1. Install itchysats service
2. configure, startup
3. create backup
4. uninstall
5. delete browser cookies for `embacy-..-.local
6. refresh the tab, login
7. restore from backup
8. The `LAUNCH UI` link is still incorrect
9. Switched to Chrome, login, same result

`Clearing cookies and site data` has not resolved the issue. Is that what you meant @islandbitcoin ?
I don't think it's a problem of resolving the link, but the link is actually incorrect.

I wonder if that is related to backing up the app's config - at the moment I only back up the `/data` folder with the app data, but not the config.

**Comment by da-kami** [comment]
Here's a video that shows the whole process and the problem: 

https://drive.google.com/file/d/1IgRGCJAlP4g7_2eXxOIkAGSJSBZxeEuy/view?usp=sharing

Tried to record it in a GIF but it exceed GitHub's max file size of 10MB, so link to GDrive after all... 😬

**Comment by da-kami** [comment]
So, part of the problem was that the `main` volume was not backed up. I fixed that, but the problem persisted.
The problem was not related to the browser cache, resetting cookies, hard refresh, switching to different browser did not help.

---

Eventually I noticed that I still kept seeing the `Temporary failure in name resolution` problem that I reported in #3.
I noticed that this problem actually affects the whole system, somehow the nameserver is not configured correctly which results

This not only has an affect on `ItchySats`, but on the whole system. Notably, `bitcoind` eventually started running into sync problems because the reported system time was incorrect (i.e. in the past) because due to not being able to properly resolve names the raspi was unable to sync with the time server. 

Note: I had ran into the `Temporary failure in name resolution` problem when using the embassy with multiple different routers. I don't exactly understand what is causing this issue. I did not have such kind of issues with my Umbrel node (or any other of my raspberries).

---

Eventually I was able to overcome this problem by manually configuring a nameserver on the embassy:

```
sudo sh -c 'echo "nameserver 8.8.8.8" >> /etc/resolv.conf'
```

Once DNS was properly working again all the problems went away. Bitcoind worked properly again and I could resolve the service URL.
I tried to configure the nameserver more permanently by installing `resolveconf` (as described in [this tutorial](https://www.tecmint.com/set-permanent-dns-nameservers-in-ubuntu-debian/)) but somehow `/etc/resolv.conf` does not reflect the configured nameserver. 

Does the embassy configure it's own DNS that overwrites `/etc/resolv.conf`?
What could cause this problem?
